h2o-core/src/main/java/water/rapids/PermutationVarImp.java

```diff
@@ -0,0 +1,216 @@
+                Vec origFeature = fr.replace(f, shuffledFeature);
+
+                // score the model again and compute diff
+                _model.score(fr).remove();
+
+                // save the difference for the given variable
+                result.put(variables[f], Math.abs(getMetric(ModelMetrics.getFromDKV(_model, fr), metric) - origMetric));
+
+                // return the original data
+                fr.replace(f, origFeature);
+
+                shuffledFeature.remove();
+                shuffledFeature = null;
+            }
+        } finally {
+            if (null != fr && fr != _inputFrame) fr.remove();
+            if (null != shuffledFeature) shuffledFeature.remove();
+        }
+        return result;
+    }
+
+    /**
+     * Get PermutationVarImp
+     *
+     * @param metric    Metric to use to calculate the feature importance
+     * @param n_samples Number of samples to use to calculate the feature importance; Use -1 to use the whole frame
+     * @param features  Features to evaluate
+     * @param seed      Seed for random generator
+     * @return TwoDimTable of Permutation Feature Importance scores
+     */
+    public TwoDimTable getPermutationVarImp(String metric, final long n_samples, final String[] features, long seed) {
+        metric = inferAndValidateMetric(metric);
+
+        HashMap<String, Double> varImps = calculatePermutationVarImp(metric, n_samples, features, seed);
+
+        String[] names = new String[varImps.size()];
+        double[] importance = new double[varImps.size()];
+
+        int i = 0;
+        for(Map.Entry<String, Double> entry : varImps.entrySet()) {
+            names[i] = entry.getKey();
+            importance[i++] = entry.getValue();
+        }
+
+        // Create TwoDimTable having (Relative + Scaled + percentage) importance 
+        return ModelMetrics.calcVarImp(importance, names);
+    }
+
+    /**
+     * Get Repeated Permutation Variable Importance
+     *
+     * @param metric    Metric to use to calculate the feature importance
+     * @param n_samples Number of samples to use to calculate the feature importance; Use -1 to use the whole frame
+     * @param n_repeats Number of repeats
+     * @param features  Features to evaluate
+     * @param seed      Seed for random generator
+     * @return TwoDimTable of Permutation Feature Importance scores
+     */
+    public TwoDimTable getRepeatedPermutationVarImp(String metric, final long n_samples, final int n_repeats ,final String[] features, long seed) {
+        metric = inferAndValidateMetric(metric);
+
+        HashMap<String, Double>[] varImps =  new HashMap[n_repeats];
+        for (int i = 0; i < n_repeats; i++) {
+            varImps[i] = calculatePermutationVarImp(metric, n_samples, features, (seed == -1 ? -1 : seed + i));
+        }
+
+        String[] names = new String[varImps[0].size()];
+        double[/*r*/][/*c*/] importance = new double[varImps[0].size()][n_repeats];
```

Comment:
This could not be very clear. I will rather use whole words if the comments are necessary. 

```suggestion
        double[/*repeat*/][/*colums*/] importance = new double[varImps[0].size()][n_repeats];
```